scalafix-core/shared/src/main/scala/scalafix/internal/util/PrettyType.scala

```diff
@@ -185,32 +207,24 @@ class PrettyType private (table: SymbolTable, shorten: QualifyStrategy) {
               toType(ret)
             )
           }
-        case t.CLASS_INFO_TYPE =>
-          val classInfo = info.typ.classInfoType.get
+        case s.ClassSignature(Some(tparams), parents, self, Some(decls)) =>
           val declarations =
-            classInfo.declarations.iterator.filter(sym =>
-              !sym.contains("$anon"))
-          val typeParameters = classInfo.typeParameters
-          val parents = classInfo.parents
+            decls.infos.filter(info => !info.symbol.contains("$anon"))
```

Comment:
Is this filter still necessary after all the fixes that we did around 4.0.0-M2?

Reply:
Good question. Doesn't seem required anymore, I removed it.